platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -465,21 +465,38 @@ influential.
 
 **Platforms** enable developers and operators to deliver applications and
 services faster by providing and managing common capabilities. Platforms bridge
-between platform users and platform capability providers.
+between platform users and platform capability providers, and are built and
+maintained by platform teams.
+
+**Platform capability providers** develop and maintain the capabilities offered
+by the platform. Providers can be both external organizations or internal teams,
+and capabilities can be infrastructure, runtime, or other supporting services.
+
+**Platform developers** and **platform engineers** are responsible for developing and 
```

Comment:
Are these two terms used as synonyms or is there a difference we are trying to convey? Could we keep only one of the two?

Reply:
I found both terms in the paper, and I decided to keep them both here as well for the sake of consistency. The term "engineers" is more inclusive as it includes also people who don't write code, while "developers" is more code-related.

We can choose to adopt only one of them: in this case, I suggest reviewing the entire paper to apply the change consistently. I can take care of this if needed.

Reply:
Both work depending on the organizations structure, but I would agree that using engineer is a good default to use.

Reply:
I would also lean towards "platform engineers" and use that consistently throughout the paper.

Reply:
+1 from me, might as well use "platform engineer" here in line with the industry buzz. I think it's okay to use "developer" as a synonym in other places in the doc though to keep it interesting.